x/nexus/keeper/keeper.go

```diff
@@ -248,6 +265,25 @@ func (k Keeper) GetTransfersForChain(ctx sdk.Context, chain exported.Chain, stat
 	return transfers
 }
 
+func (k Keeper) getPendingTransfersForRecipient(ctx sdk.Context, recipient exported.CrossChainAddress) []exported.CrossChainTransfer {
```

Comment:
```suggestion
func (k Keeper) getPendingTransferForRecipientAndAsset(ctx sdk.Context, recipient exported.CrossChainAddress, denom string) exported.CrossChainTransfer {
```
There's should be only one anyway.

Reply:
done